### PR TITLE
feat: Added Product details page

### PR DESCRIPTION
**Pull request by TusharJade**
Added product details page 
LIve link :

### CODE REVIEW

File: src/components/ProductCard/ProductCard.css [hunk 1]
```diff
@@ -38,4 +39,9 @@
 
 .price{
     margin-top: 0.3rem;
-}
\ No newline at end of file
+}
+
+.cart-link{
+    text-decoration: none;
+    color: black;
```
Comment (sakshi006) [inline src/components/ProductCard/ProductCard.css:46]: use CSS variable

File: src/components/ProductCard/ProductCard.jsx [hunk 2]
```diff
@@ -28,14 +34,21 @@ const ProductCard = ({ item }) => {
 
       {cartState.cartList.find((cartItem) => cartItem._id === item._id) ? (
         <button className="card-add-btn">
-          <Link to="/AddToCart" className="link">
+          <Link
+            to="/AddToCart"
+            className="link"
+            onClick={(e) => e.stopPropagation()}
```
Comment (sakshi006) [inline src/components/ProductCard/ProductCard.jsx:41]: Oh, this stopPropagation is new for me!

File: src/pages/ProductDetailsPage/ProductDetailsPage.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+
+
```
Comment (sakshi006) [inline src/pages/ProductDetailsPage/ProductDetailsPage.css:2]: i think you can remove this file?

**Review by sakshi006 — APPROVED** [review]
looks good to merge

File: src/components/ProductCard/ProductCard.css (hunk 1 above)
Comment (TusharJade) [reply src/components/ProductCard/ProductCard.css:46]: Yes forgotten that will chnge it now

File: src/components/ProductCard/ProductCard.jsx (hunk 2 above)
Comment (TusharJade) [reply src/components/ProductCard/ProductCard.jsx:41]: For me too, just found that today in someone else code it's so useful.

File: src/pages/ProductDetailsPage/ProductDetailsPage.css (hunk 3 above)
Comment (TusharJade) [reply src/pages/ProductDetailsPage/ProductDetailsPage.css:2]: yes will remove it, btw thanks for reviewing code in detail your review are so helpful 🙌

**Review by LordUttam — APPROVED** [review]
Looks good, go ahead.

**Pull request merged**